todo_app/app.py

```diff
@@ -2,26 +2,36 @@
 from todo_app.data.todo_app_items import get_items_all, add_item, close_item
 from todo_app.flask_config import Config
 from todo_app.data.view_model import ViewModel
-from dotenv import load_dotenv
+from loggly.handlers import HTTPSHandler
+from logging import Formatter
 
 def create_app():
 
     app = Flask(__name__)
     app.config.from_object(Config())
-
+    app.logger.setLevel(app.config['LOG_LEVEL'])
+    if app.config['LOGGING_SERVICE_TOKEN'] is not None:
+        handler = HTTPSHandler(f'https://logs-01.loggly.com/inputs/{app.config["LOGGING_SERVICE_TOKEN"]}/tag/todo-app')
+        handler.setFormatter(Formatter("[%(asctime)s] %(levelname)s in %(module)s: %(message)s"))
+        app.logger.addHandler(handler)
+    
     @app.route('/')
     def index():
         var_view_model = ViewModel(get_items_all())
         return render_template('index.html', html_view_model = var_view_model )
 
     @app.route('/additem', methods=['POST'])
     def additem():
-        add_item(request.form.get('title'))
+        log_add_item = request.form.get('title')
+        add_item(log_add_item)
+        app.logger.info('The following Item has been added: %s', log_add_item)
```

Comment:
Minor here but always worth considering; if this was real public user data, would this log potentially lead to you logging personal data in production? It seems very plausible to me we might get to-dos like:
* Buy <X> their 50th birthday present
* Send <email_address> a message about <thing>
So while this is a really natural message, maybe we'd want to log an id after creating the item or similar, rather than the content of the message itself

Reply:
I agree. 
Looking at the code, I think I need to write a get_id function first as I can't see a way with the current code base to do this? But normally this is where you point out to me I've missed an easy trick somewhere?

Reply:
Aha, you've already done the hard work actually!

Your `add_item` function ends with:
```python
    ai_item = posts.insert_one(post).inserted_id
    
    return ai_item
```
So you already find the id and return it, we could just print here with:
```
        id = add_item(log_add_item)
        app.logger.info('The following Item has been added: %s', id)
```

Reply:
I knew it lol :)
Thank you!